README.md

```diff
@@ -1,2 +1,238 @@
-# manyclangs
-Repository hosting unofficial binary pack files for many commits of llvm
+# manyclangs <img src="https://llvm.org/img/DragonSmall.png" width="auto" height="72">
+manyclangs is a project enabling you to run any commit of clang within a few seconds, without having to build it.
+
+It provides [elfshaker](https://github.com/elfshaker/elfshaker) pack files, each containing ~2000 builds of LLVM packed into ~100MiB. Running any particular build takes about 4s.
+
+## Quickstart
+
+### AArch64
+Requirements: `elfshaker`, `wget`, `git`
+```bash
+mkdir -p manyclangs/elfshaker_data/packs
+YYYYMM=202109
+wget https://github.com/elfshaker/manyclangs/releases/download/v0.9.0/aarch64-ubuntu2004-manyclangs-$YYYYMM.pack{,.idx} -P manyclangs/elfshaker_data/packs
+cd manyclangs
+elfshaker find ''
```

Comment:
`find ''`?

Reply:
Oh, right, it is going to match all snapshots. Nice. Surprising somehow, but OK :)

Reply:
No, it is on purpose. We've discussed it before.

Reply:
It just looks surprising, I think, in user interface terms. I had forgotten, and wondered if it was the document rendering incorrectly somehow, for a moment.